lighthouse-core/lib/lh-error.js

```diff
@@ -47,6 +47,11 @@ const UIStrings = {
   internalChromeError: 'An internal Chrome error occurred. Please restart Chrome and try re-running Lighthouse.',
   /** Error message explaining that fetching the resources of the webpage has taken longer than the maximum time. */
   requestContentTimeout: 'Fetching resource content has exceeded the allotted time',
+  /**
+   * @description Error message explaining that the webpage is non-HTML, so audits are ill-defined.
+   * @example {application/xml} mimeType
+   * */
+  nonHtml: 'The page provided is not HTML (served as MIME type {mimeType}).',
```

Comment:
nit: `notHtml` instead? and `NOT_HTML`

Reply:
I kept the nonHtml name scheme from the function but this makes sense since the function isn't really in context here, I'll change this!